airflow-core/src/airflow/example_dags/example_asset_alias.py

```diff
@@ -34,6 +34,7 @@
 
 import pendulum
 
+# [START example_asset_alias]
```

Comment:
Marker doesn't cover the pendulum import

Reply:
Changed it.